### PR TITLE
📖 Add test trigger description for diffrerent branches in README

### PR DESCRIPTION
**Pull request by kashifest**
As the title says.

### CODE REVIEW

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Comment by kashifest** [comment]
cc @dtantsur @honza

File: README.md [hunk 1]
```diff
@@ -28,6 +28,27 @@ components, see the [Metal³ docs](https://github.com/metal3-io/metal3-docs).
...
+* **/test-centos-e2e-integration-main** runs CAPM3 e2e integration tests with
+  CAPM3 main branch and BMO **main** branch on Centos
+* **/test-ubuntu-integration-main** runs ansible integration tests with CAPM3
+  main branch and BMO **main** branch on Ubuntu
+
+Release-0.4 branch:
+
+* **/test-centos-e2e-integration-release-1-5** runs CAPM3 e2e integration tests
+  with CAPM3 release-1.5 branch and BMO **release-0.4 branch** on Centos
+* **/test-ubuntu-integration-main** runs ansible integration tests with CAPM3
```
[the remark this answers is not in the document]
Comment (kashifest) [reply README.md]: copy-paste effect.

**Comment by kashifest** [comment]
/override /test-ubuntu-integration-main
/override /test-centos-e2e-integration-main
Its a README change only.

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Comment by kashifest** [comment]
/override test-ubuntu-integration-main
/override test-centos-e2e-integration-main
Thanks @lentzi90

**Pull request merged**